src/main/java/ReviewResponse.java

```diff
@@ -1,10 +1,13 @@
 import lombok.Data;
 import lombok.Getter;
+import lombok.Setter;
```

Comment:
Using Lombok annotations like @Setter can lead to potential issues with encapsulation and violate the principle of data hiding. (Severity: medium)